docs/book_files/csi_block_storage_kc_pdfs.md

```diff
@@ -0,0 +1,40 @@
+# Publications and related information
+
+The following Portable Document Format (PDF) publications and related online information sources are available for the IBM® block storage CSI driver.
+
+To view a PDF file, you need Adobe™ Reader. You can download it at no charge from the [Adobe website](http://get.adobe.com/reader/).
+
+## Publications
+
+|IBM block storage CSI driver publication|Description|PDF download link|
+|----------------------------------------|-----------|-----------------|
+|*Release notes*|This publication describes requirements, compatibility, change log, and known issues information for version 1.6.0.<br />First Edition (June 2021) for version 1.6.0.|![PDF icon](PDF_icon.jpg) [Download](../pdf/IBM_block_storage_CSI_driver_1.6.0_RN.pdf)|
+|*User guide*|This publication describes how to prepare for, install, configure, and use IBM block storage CSI driver.<br />Publication number: SC27-9590-10.|![PDF icon](PDF_icon.jpg)[Download](../pdf/IBM_block_storage_CSI_driver_1.6.0_UG.pdf)|
```

Comment:
```suggestion
|*User guide*|This publication describes how to prepare for, install, configure, and use IBM block storage CSI driver.<br />Publication number: SC27-9590-10.|![PDF icon](PDF_icon.jpg) [Download](../pdf/IBM_block_storage_CSI_driver_1.6.0_UG.pdf)|
```
as in the row above

Reply:
updated in source files. will be reflected in next push.

Reply:
when is the next push?

Reply:
@zingero it should already be reflecting

Reply:
![image](https://user-images.githubusercontent.com/9675399/121870834-1a54d180-cd0c-11eb-906c-5a6d3667d851.png)

I see a missing whitespace between the icon and the text

Reply:
this is now fixed in IBM Docs